src/main/java/io/cdap/plugin/http/sink/batch/HTTPRecordWriter.java

```diff
@@ -131,7 +134,7 @@ public void write(StructuredRecord input, StructuredRecord unused) throws IOExce
 
     if (config.getMethod().equals(REQUEST_METHOD_PUT) || config.getMethod().equals(REQUEST_METHOD_PATCH) ||
       config.getMethod().equals(REQUEST_METHOD_DELETE)
-      && !placeHolderList.isEmpty()) {
+        && !placeHolderList.isEmpty()) {
```

Comment:
revert the changes due to indentation as it makes things confusing for the reviewer. In actions on save, reformat the file only where you are making changes

Reply:
reverted